tests/docker-compose.auth-server.yml

```diff
@@ -0,0 +1,47 @@
+version: '3.3'
```

Comment:
I believe right now we don't specify `version`. Please see the PR from @SpecLad .